misk/src/main/kotlin/misk/web/extractors/QueryParamFeatureBinding.kt

```diff
@@ -31,7 +31,9 @@ internal class QueryParamFeatureBinding private constructor(
     private val name: String
   ) {
     fun bind(subject: Subject) {
-      val values = subject.httpCall.url.queryParameterValues(name)
+      val values = subject.httpCall.url.queryParameterValues(name).map {
+        it ?: throw IllegalArgumentException()
```

Comment:
this is correct but I think we later want to make a decision on what we do for `b` in `https://host/path?a=1&b&c=2`